src/app/services/currentLocationsData.service.ts

```diff
@@ -9,19 +9,38 @@ export class CurrentLocationsService {
   constructor(private _jsonp: Jsonp) {}
 
   getData(position: Position): Observable<IDataFromDto> {
-      const centerPoint = `${position.coords.latitude},${position.coords.longitude}`;
+    const countryKey = localStorage.getItem(COUNTRY_NAME_KEY);
 
-      const countryKey = localStorage.getItem(COUNTRY_NAME_KEY);
-      const url = `https://api.nestoria.${countryKey}/api?pretty=1&action=search_listings&encoding=json&` +
-        `listing_type=buy&page=1&callback=JSONP_CALLBACK&centre_point=${centerPoint}`;
+    const params = this.getParams(position);
```

Comment:
const params = this.httpUtils.getQuery({
position: position,
pretty: 1,
...
});